cmd/infracost/run.go

```diff
@@ -226,36 +153,23 @@ func runMain(cmd *cobra.Command, runCtx *config.RunContext) error {
 
 	r.RunID, r.ShareURL = result.RunID, result.ShareURL
 
-	opts := output.Options{
-		DashboardEnabled: runCtx.Config.EnableDashboard,
-		ShowSkipped:      runCtx.Config.ShowSkipped,
-		NoColor:          runCtx.Config.NoColor,
-		Fields:           runCtx.Config.Fields,
-	}
-
-	var b []byte
-
 	format := strings.ToLower(runCtx.Config.Format)
-	if runCtx.Config.CompareTo != "" && !validCompareToFormats[format] {
+	isCompareRun := runCtx.Config.CompareTo != ""
+	if isCompareRun && !validCompareToFormats[format] {
 		return errors.New("The --compare-to option cannot be used with table and html formats as they output breakdowns, specify a different --format.")
 	}
 
-	switch format {
-	case "json":
-		b, err = output.ToJSON(r, opts)
-	case "html":
-		b, err = output.ToHTML(r, opts)
-	case "diff":
-		b, err = output.ToDiff(r, opts)
-	default:
-		b, err = output.ToTable(r, opts)
-	}
-
+	b, err := output.MarshalOutput(format, r, output.Options{
```

Comment:
```suggestion
	b, err := output.FormatOutput(format, r, output.Options{
```
Being picky here, but only because marshal to me means serializing (i.e. converting to a machine format).

Reply:
👍